packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsEditBox.tsx

```diff
@@ -221,7 +227,11 @@ export const ChatMessageComponentAsEditBox = (props: ChatMessageComponentAsEditB
     <ChatMyMessage
```

Comment:
ChatMyMessage is v9 component, `mergeStyles` is v8 styles api, `chatMyMessageStyles` is v9 styles. Please use `mergeClasses` in this case instead to use v9 styles api with v9 component

Reply:
good catch